dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataOwnershipRequestEmailBuilder.kt

```diff
@@ -0,0 +1,61 @@
+package org.dataland.datalandbackend.services
+
+import org.dataland.datalandbackendutils.email.BaseEmailBuilder
```

Comment:
What iam asking myself is, is the backend the right place for this class? Or should we bundle all e-mail related stuff into the same place? Maybe we could even generalize the different e-mail templates

Reply:
Hmm. I see. There is not much left from the original class. And the inheriting classes look basically the same.
I had issues with putting the functionality of buildUserInfo into the backendutils because it uses DatalandAuthentication which comes from our dataland-keycloak-adapter subproject. Which in turn depends on the backendutils. So this would result in a circular dependency. I could extract all the email stuff into a separate library. What do you think?

Reply:
Maybe we can have a short call this afternoon and see if it has merti to extract it

Reply:
I'll call you when I have found a quiet spot

Reply:
You moved this to the new libary, correct?

Reply:
Yes.

Reply:
Perfect, then this is closed